setup.py

```diff
@@ -13,9 +13,9 @@
 
 
 prod_require = [
-    'Django>=1.11,<2.3',
-    'djangorestframework>=3.11.0,<3.12',
-    'django-filter>=2.2.0,<2.3',
+    'Django>=2.2,<3.2',
+    'djangorestframework>=3.12.1,<3.13',
```

Comment:
do we need to restrict upper version bounds?

Reply:
I don´t think so. I only followed the existing spelling ^^ I´ll remove it